src/main/java/uk/gov/hmcts/sptribs/ciccase/model/CaseData.java

```diff
@@ -425,6 +426,14 @@ public class CaseData {
     @JsonUnwrapped
     private RetiredFields retiredFields;
 
+    @CCD(
+        label = "State to transition test case to",
+        typeOverride = FixedList,
+        typeParameterOverride = "State",
+        access = {DefaultAccess.class}
+    )
+    private State stateToTransitionApplicationTo;
```

Comment:
Is it possible to avoid introducing new fields for testing purpose only?

Reply:
Have updated using existing fields